sotodlib/toast/workflows/proc_demodulation.py

```diff
@@ -38,12 +38,14 @@ def setup_demodulate(operators):
             nbin_psd=64,
             nsum=1,
             naverage=64,
+            enabled=False,
         )
     )
     operators.append(
         toast.ops.FitNoiseModel(
             name="demod_noise_estim_fit",
             out_model="demod_noise_fit",
+            enabled=False,
```

Comment:
For the noise estimation and fitting "sets of operators", I had been disabling the "estimation" operator by default and leaving the "fitting" and "flagging" operators enabled.  This allows the user to just enable the estimation operator and then the fitting / flagging will just happen.  The raw noise estimates are usually too jagged to use directly as a noise model / filter.  [Here is the workflow logic for cutting noise outliers](https://github.com/simonsobs/sotodlib/blob/304ff59c87203faba7a427a2a532c43a07fc1b08/sotodlib/toast/workflows/proc_flagging.py#L107-L117) and [here is the logic](https://github.com/simonsobs/sotodlib/blob/304ff59c87203faba7a427a2a532c43a07fc1b08/sotodlib/toast/workflows/proc_noise_est.py#L65-L69) for running noise estimation for mapmaking.